recipes/bisonpp/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "bisonpp" %}
+{% set version = "1.21-45" %}
+{% set sha256 = "0b72facde5587d9908077ec16868f404f3d879902c15360754349ac17a8f1129" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/jarro2783/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+requirements:
+  build:
+    - gcc
```

Comment:
Can you switch to the new compilers please

Reply:
sure! thanks!